src/adaptor.py

```diff
@@ -322,7 +317,8 @@ def bootstrap():
 
     args = parser.parse_args()
 
-    main.setvar(env=args.instance_id)
+    # TODO: how to pass this to render?
+    # main.setvar(env=args.instance_id)
```

Comment:
`instance_id` is used for the cdn link generation. cdn link should be read from a config file and this `instance_id` argment should be removed